plenoptic/synthesize/mad_competition.py

```diff
@@ -2097,6 +2097,18 @@ def animate(self, synthesis_target=None, batch_idx=0, channel_idx=None,
             The animation object. In order to view, must convert to HTML
             or save.
 
+        Notes
```

Comment:
not sure how I feel about all these being duplicated across diff files

Reply:
Yeah, that's fair. This is a symptom of a broader issue we've talked about -- with the inheritance of methods across `Synthesis` means the docstrings get duplicated or changed slightly. We have an issue for it, #25, and the solution seaborn uses is to have string constants containing the info that can then be inserted into the docstring. We could check other libraries to see how they handle it too. 

I don't have a problem with the info showing up in multiple places (in fact, I think it should -- the user should see the info where it's relevant to them, though I'd also be fine with a brief discussion in most docstrings, pointing to a "main" one with longer discussion), but it makes it hard to keep everything up-to-date